projects/storefrontlib/cms-components/content/tab/panel/tab-panel.component.html

```diff
@@ -0,0 +1,25 @@
+<div
+  role="tabpanel"
+  [attr.aria-labelledby]="tab?.id"
+  [attr.id]="'section-' + tabNum"
+  [attr.tabindex]="isOpen ? 0 : -1"
+  [class.active]="isOpen"
+  [style.display]="isOpen ? 'block' : 'none'"
+  [cxFocus]="{ focusOnEscape: true }"
+  [ngSwitch]="getContentType()"
```

Comment:
Could we dumb this down to only render TemplateRefs? This could prevent trying to catch all content-type cases from within this class for compatibility when it might be better to handle this case-by-case (ie. convert any content into templateRef before passing in TabParagraphComponent, Facets, etc).